### DISCUSSION TITLE
When will version 0.7.0 be released?

### DISCUSSION THREAD

**Discussion by filwaline** [mdapena/pyventus#27] [General]
Version 0.7.0 seems to have some break changes. I want to integrate this package with my new project. If it still takes a long time, I may just move forward with version 0.6.0.

**Comment by mdapena** [comment]
Hi again @filwaline,

Thanks for your interest in the project and the upcoming release!

Unfortunately, Pyventus v0.7.0 won't be released until the minimal documentation is completed, as it is very important for users to understand how to effectively utilize the library. While all the new features are implemented and tested, it may take me some time to finish the documentation since I’m working on this project in my free time and have been quite busy lately :(

In the meantime, you can definitely use version 0.6.0 without any issues. The breaking changes in the upcoming release are mainly related to method renames in the EventLinker and some module imports. These can be easily fixed using auto-fix tools, so you should be good to go!

I’m aiming for a release in January or February next year, and I’ll keep this discussion updated with any news.

Let me know if you have any other questions!

**Comment by mdapena** [comment]
Hello everyone, and Happy New Year! 🎉

Just a quick update here to share that Pyventus `v0.7.0` is now live! You can check out the [Introducing Pyventus v0.7.0 🚀](https://github.com/mdapena/pyventus/discussions/33) announcement for an overview of the release. I hope you all enjoy it!

Happy coding!
